### ISSUE TITLE
Add warning for all Russian speaking drivers in Goa, India

### ISSUE THREAD

**Issue by ro31337** [ro31337/libretaxi#600]
Story about region specific condition has been delivered at https://github.com/ro31337/libretaxi/issues/597.

For the sake of experiment, and as a real feature we want, add warning to all Russian speaking drivers (because there are most of them) in Goa, India.  Message should appear at check-in while setting up LibreTaxi as a driver.

Message should be posted on [Telegraph](https://telegra.ph) and will be provided for this story soon.

(message to English speaking drivers will be added in another story)

**Comment by ro31337** [comment]
Link: http://telegra.ph/Voditel-v-Goa-Prochti-08-12

**Comment by ro31337** [comment]
Depends on https://github.com/ro31337/libretaxi/issues/601

**Comment by ro31337** [comment]
#601 was implemented, unblocked here.

**Comment by ro31337** [comment]
Implemented

![image](https://user-images.githubusercontent.com/1477672/30771105-96b18938-9ff4-11e7-8873-90938a23ea66.png)
